app/assets/javascripts/models/data/collection.js.coffee

```diff
@@ -259,7 +256,7 @@ ns.Collection = do (ko
       null
 
     canFocus: ->
-      @hasAtomData() && @granuleDatasourceName()
+      @hasAtomData() && @granuleDatasourceName()?
```

Comment:
Without the "?", canFocus will not always be eval to a boolean value. Therefore the granule filter button will not show up for cwic collection.